data/db-schema/src/main/resources/net/corda/db/schema/vnode-vault/migration/ledger-utxo-creation-v5.1.xml

```diff
@@ -15,6 +15,94 @@
                 tableName="utxo_transaction_output">
             <column name="type"/>
         </createIndex>
+
+    <property name="json.column.type" value="JSONB" dbms="postgresql"/>
+
+    <!-- Please note that HSQLDB schema is supported for integration test purposes only. -->
+    <property name="json.column.type" value="CLOB" dbms="hsqldb"/>
+
+    <changeSet author="R3.Corda" id="ledger-utxo-creation-v5.2">
+
+        <!-- Add new status column to utxo_transaction -->
+        <addColumn tableName="utxo_transaction">
+            <column name="status" type="VARCHAR(255)">
```

Comment:
There were some discussions to change this to CHAR(1) or something similar since we put only 1 letter there.

Reply:
I believe `VARCHAR` will only use the space needed based on the actual length of the string, so this probably doesn't need to be exactly right. but ideally it would be in the right ballpark. I'd imagine this will only be 1-2 characters in practice even taking future statuses into account, so something like 16/32 would be more reasonable IMO.

(FWIW, I believe `CHAR` _can_ be slightly more efficient to index on etc. since it's always a fixed length, but I don't think that is worth worrying about right now).